spec/features/admin/schools/changing_provision_spec.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.feature "Admin managing school provision", js: true, rutabaga: false do
+  scenario "Admin changes school provision" do
+    given_there_is_a_fip_school_in_2021
+    and_i_am_signed_in_as_an_admin
+    when_i_visit_the_school_cohorts_page
+    and_i_click_the_change_link
+    then_i_should_be_on_the_change_programme_page
+    and_the_page_should_be_accessible
+    and_percy_should_be_sent_a_snapshot_named("Admin choose programme page")
+
+    when_i_select_cip
+    and_i_click_the_submit_button
+    then_i_should_be_on_the_confirm_page
+    and_the_page_should_be_accessible
+    and_percy_should_be_sent_a_snapshot_named("Admin confirm programme page")
+
+    when_i_click_the_submit_button
+    then_i_should_be_on_the_school_cohorts_page
+    and_there_should_be_a_success_banner
+  end
+
+private
+
+  def given_there_is_a_fip_school_in_2021
+    cohort = create(:cohort, start_year: 2021)
+    @school_cohort = create(:school_cohort, cohort: cohort, induction_programme_choice: "full_induction_programme")
+  end
```

Comment:
Not sure if this is the right PR for it, but I would love for us to have a similar shared steps setup to what we have in cypress tests.

Reply:
I've written all the feature steps inline at the moment. I intend to extract them as they're reused